packages/notification-config-process/.env

```diff
@@ -11,7 +11,7 @@ EVENTSTORE_DB_SCHEMA=notification_config
 EVENTSTORE_DB_USE_SSL=false
 
 WELL_KNOWN_URLS="http://127.0.0.1:4500/jwks.json"
-ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui,dev.interop.pagopa.it/ui,dev.interop.pagopa.it/internal,dev.interop.pagopa.it/m2m,dev.interop.pagopa.it/m2m"
+ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui"
```

Comment:
edited this to remove duplicates and audiences that aren't needed (at least for now – internal might be needed for configurations created by the onboarding process)